packages/oas-resolver/package-lock.json

```diff
@@ -13,155 +13,83 @@
 			}
 		},
 		"ansi-regex": {
-			"version": "3.0.0",
-			"resolved": "https://registry.npmjs.org/ansi-regex/-/ansi-regex-3.0.0.tgz",
-			"integrity": "sha1-7QMXwyIGT3lGbAKWa922Bas32Zg="
+			"version": "4.1.0",
+			"resolved": "https://packages.atlassian.com/api/npm/npm-remote/ansi-regex/-/ansi-regex-4.1.0.tgz",
```

Comment:
Is there a reason your `package-lock.json` changes are resolving against `packages.atlassian.com` instead of the canonical `npm` registry?

This is one reason I generally do not accept PRs which update `package-lock.json` as it's a potential vector for security issues itself.

Reply:
OMG, I'm so sorry, yeah, that should not have been the case, if not for my local setup which was dictated by company policy of which I completely forgot.